indico/modules/events/abstracts/templates/display/_base.html

```diff
@@ -1,4 +1,16 @@
 {% extends 'layout/conference_page_base.html' %}
+{% block page_actions %}
+    <a href="{{ url_for('.my_abstracts', abstract.event_new) }}" id="button-back" class="i-button">
+        Back to My Abstracts</a>
+    <script>
+        var backUrl =  $.jStorage.get('back-url');
```

Comment:
this creates a global variable... wrap the whole thing in a strictmode function:

    <script>
        (function(){
            'use strict';

            ...
        })();
    </script>